### ISSUE TITLE
Unable to use rollup.js with protobuf-ts due to circular dependency

### ISSUE THREAD

**Issue by taylorcode** [timostamm/protobuf-ts#269]
**Versions:**

> @protobuf-ts/runtime@2.4.0
> @protobuf-ts/runtime-rpc@2.4.0
> @protobuf-ts/plugin@2.4.0

rollup.js fails to bundle my project because of a circular dependency inside of the @protobuf-ts/runtime package.

1. `npx protoc --ts_out` generates code with these imports:
```
// @generated by protobuf-ts 2.4.0,// @generated from protobuf file "msg-readme.proto" (syntax proto3),// tslint:disable
import { ServiceType } from "@protobuf-ts/runtime-rpc";
import { MessageType } from "@protobuf-ts/runtime";
```
3. My project imports this generated code

4. Rollup attempts to bundle my project's code, and fails with this error:

```
{
  code: 'CIRCULAR_DEPENDENCY',
  cycle: [
    'node_modules/@protobuf-ts/runtime/build/es2015/binary-format-contract.js',
    'node_modules/@protobuf-ts/runtime/build/es2015/binary-reader.js',
    'node_modules/@protobuf-ts/runtime/build/es2015/binary-format-contract.js'
  ],
  importer: 'node_modules/@protobuf-ts/runtime/build/es2015/binary-format-contract.js',
  message: 'Circular dependency: node_modules/@protobuf-ts/runtime/build/es2015/binary-format-contract.js -> node_modules/@protobuf-ts/runtime/build/es2015/binary-reader.js -> node_modules/@protobuf-ts/runtime/build/es2015/binary-format-contract.js',
}
```

Upon investigating, I see that rollup is indeed correct.

**binary-format-contract.js** contains this import:
```
import { BinaryReader } from "./binary-reader";
```

and **binary-reader.js** contains this import:
```
import { WireType } from "./binary-format-contract";
```

**Comment by timostamm** [comment]
Thanks for the report, Taylor. 

Looks like `binaryReadOptions()` and `binaryWriteOptions()` should simply move to `binary-reader.ts` / `binary-writer.ts`.

**Comment by timostamm** [comment]
Fixed in https://github.com/timostamm/protobuf-ts/pull/271. 
Would appreciate if you could try this or give it a :eyes:.

**Comment by taylorcode** [comment]
yes this fixes it :) just verified it locally.

thank you for responding to this issue so fast!

Can you publish this update to NPM so I can avoid creating a fork?

Also, it looks like the rollup build does in fact finish and produce the correct output bundle, but my build system (Bazel) which calls rollup is configure to exit on unexpected warnings.

**Comment by timostamm** [comment]
The fix is released in [v2.5.0](https://github.com/timostamm/protobuf-ts/releases/tag/v2.5.0).

**Comment by taylorcode** [comment]
awesome!
